index.html

```diff
@@ -0,0 +1,230 @@
+              EPUB
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-sm-6 col-md-4">
+        <div class="card mb-3">
+          <img class="card-img-top" src="/assets/images/cover-virtualization.svg">
+          <div class="card-body">
+            <h5 class="card-title">
+              {{ locale.virtualization_guide }}
+            </h5>
+            <a class="card-link"
+              href="https://doc.opensuse.org/documentation/leap/archive/{{ leap.version }}/virtualization/html/book.virt/index.html">
+              HTML
+            </a>
+            <a class="card-link"
+              href="https://doc.opensuse.org/documentation/leap/archive/{{ leap.version }}/virtualization/single-html/book.virt/index.html">
+              Single HTML
+            </a>
+            <a class="card-link"
+              href="https://doc.opensuse.org/documentation/leap/archive/{{ leap.version }}/virtualization/book.virt_color_en.pdf">
+              PDF
+            </a>
+            <a class="card-link"
+              href="https://doc.opensuse.org/documentation/leap/archive/{{ leap.version }}/virtualization/book.virt_en.epub">
+              EPUB
+            </a>
+          </div>
+        </div>
+      </div>
+
+      {% if leap.autoyast%}
+      <div class="col-sm-6 col-md-4">
+        <div class="card mb-3">
+          <img class="card-img-top" src="/assets/images/cover-autoyast.svg">
+          <div class="card-body">
+            <h5 class="card-title">
+              {{ locale.autoyast_guide }}
+            </h5>
+            <a class="card-link"
+              href="https://doc.opensuse.org/documentation/leap/archive/{{ leap.version }}/autoyast/html/book-autoyast/index.html">
+              HTML
+            </a>
+            <a class="card-link"
+              href="https://doc.opensuse.org/documentation/leap/archive/{{ leap.version }}/autoyast/single-html/book-autoyast/index.html">
+              Single HTML
+            </a>
+            <a class="card-link"
+              href="https://doc.opensuse.org/documentation/leap/archive/{{ leap.version }}/autoyast/book-autoyast_color_en.pdf">
+              PDF
+            </a>
+            <a class="card-link"
+              href="https://doc.opensuse.org/documentation/leap/archive/{{ leap.version }}/autoyast/book-autoyast_en.epub">
+              EPUB
+            </a>
+          </div>
+        </div>
+      </div>
+      {% endif %}
+
+      <div class="col-sm-6 col-md-4">
+        <div class="card mb-3">
+          <img class="card-img-top" src="/assets/images/cover-release.svg">
+          <div class="card-body">
+            <h5 class="card-title">
+              {{ locale.release_notes }}
+            </h5>
+            <a class="card-link"
+              href="https://doc.opensuse.org/release-notes/x86_64/openSUSE/Leap/{{ leap.version }}/RELEASE-NOTES.{{ page.locale.id | replace: '-', '_' }}.html">
```

Comment:
There is no way to make sure this works. :/ It might also work one day and fail the next, because the translation becomes outdated. The only page we really know exists is the English one. (Though I guess that might better be solvable via either a .htaccess redirect or symlinks.)

Reply:
.htaccess redirect will be a good choice